packages/dom-utilities/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "@fluentui/dom-utilities",
+  "version": "7.0.1",
+  "description": "DOM utilities for use within Fluent web components",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "start-test": "just-scripts jest-watch",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "test": "just-scripts test",
+    "start": "just-scripts dev:storybook",
+    "update-api": "just-scripts update-api"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^0.53.1",
+    "@types/jest": "~24.9.0",
+    "@types/react": "16.8.25",
+    "@types/react-dom": "16.8.4",
+    "@types/webpack-env": "1.15.1",
+    "@uifabric/build": "^7.0.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6"
+  },
+  "dependencies": {
+    "@uifabric/set-version": "^7.0.18",
+    "tslib": "^1.10.0"
+  },
+  "peerDependencies": {
+    "@types/react": ">=16.8.0 <17.0.0",
+    "@types/react-dom": ">=16.8.0 <17.0.0",
+    "react": ">=16.8.0 <17.0.0",
+    "react-dom": ">=16.8.0 <17.0.0"
+  }
```

Doesn't look like this package uses React, so unless you expect it to in the future you can remove all React stuff. You can probably also remove `@types/webpack-env`.
```suggestion
    "@uifabric/build": "^7.0.0"
  },
  "dependencies": {
    "@uifabric/set-version": "^7.0.18",
    "tslib": "^1.10.0"
  }
```